x-pack/plugins/cloud_security_posture/public/pages/configurations/findings_flyout/overview_tab.tsx

```diff
@@ -37,21 +38,28 @@ import { FindingsDetectionRuleCounter } from './findings_detection_rule_counter'
 type Accordion = Pick<EuiAccordionProps, 'title' | 'id' | 'initialIsOpen'> &
   Pick<EuiDescriptionListProps, 'listItems'>;
 
-const getDetailsList = (data: CspFinding, ruleFlyoutLink: string, discoverIndexLink?: string) => [
+const getDetailsList = (
+  data: CspFinding,
+  ruleFlyoutLink?: string,
+  discoverDataViewLink?: string
+) => [
   {
     title: i18n.translate('xpack.csp.findings.findingsFlyout.overviewTab.ruleNameTitle', {
       defaultMessage: 'Rule Name',
     }),
-    description: (
-      <EuiToolTip
-        position="top"
-        content={i18n.translate('xpack.csp.findings.findingsFlyout.overviewTab.ruleNameTooltip', {
-          defaultMessage: 'Manage Rule',
-        })}
-      >
-        <EuiLink href={ruleFlyoutLink}>{data.rule.name}</EuiLink>
-      </EuiToolTip>
-    ),
+    description:
+      ruleFlyoutLink && data.rule?.name ? (
```

Comment:
does it make sense to move to a separate component to not have this inline logic?